cdc/model/changefeed.go

```diff
@@ -137,6 +137,12 @@ type ChangeFeedInfo struct {
 	SyncPointEnabled  bool          `json:"sync-point-enabled"`
 	SyncPointInterval time.Duration `json:"sync-point-interval"`
 	CreatorVersion    string        `json:"creator-version"`
+
+	PDEndpoints   string   `json:"pd-endpoints"`
+	KeyPath       string   `json:"key-path"`
+	CertPath      string   `json:"cert-path"`
+	CAPath        string   `json:"ca-path"`
```

Comment:
What about storing cluster info in another struct and embed a pointer to that struct? Pros: When cluster info changes, we only need to change the struct not all changefeeds.